models/User.model.js

```diff
@@ -1 +1,36 @@
 // User model here
+
+const { Schema, model} = require("mongoose")
+
+const userSchema = new Schema(
+{
+    Username : {
+      type: String,
+      trim: true,
+      required: [true, 'Username is required.'],
+      unique: true
+
+    },
+    Password : {
+      type: String,
+      trim: true,
+      required: [true, 'Password is required.'],
+      },
+
+  /*     email: {
```

Comment:
It's better to remove code that doesn't serve your application to keep it small/minimal